int32 senderID
int64 syncTransitionID